.travis.yml

```diff
@@ -27,6 +27,7 @@ cache:
   - $HOME/.pip-cache
 
 install:
+  - pip install .
```

Comment:
@gpoulin is right, `usedevelop = True` (it's equivalent of `pip install -e .`) in tox.ini should do the trick.